playwright-test-js/tests/test_1.spec.js

```diff
@@ -4,10 +4,13 @@ const { expect } = require('@playwright/test')
 test.describe('Browse LambdaTest in different search engines', () => {
   test('Search LambdaTest on Bing', async ({ page }) => {
     await page.goto('https://www.bing.com')
+    await page.waitForLoadState('domcontentloaded')
```

Comment:
Why are these waits added here?

Reply:
@ankur-lt 
* domcontentloaded -> Reason to add this is, sometimes the website dom is loading but internally the xhr calls are taking more to load. 
* Added 3 secs wait -> Reason to add this is, sometimes as soon as we enter the search text the cookies popup is triggered. and it is not proceeding.

Reply:
Merging this PR. As discussed, please update the scripts accordingly in all folders.